redasql/metacommand_executor.py

```diff
@@ -242,4 +260,5 @@ def meta_command_factory(command: str):
     r'\x': ChangePivotCommandExecutor,
     r'\f': FormatterChangeExecutor,
     r'\l': LoadQueryExecutor,
+    r'\cp': ChangeCopyCommandExecutor,
```

Comment:
I'm confused about the name of the option here. I think it would be better to match it to a single letter, but it's covered with \c...

Reply:
Thanks for the PR.
I had an idea for this feature, so it's helpful.
However, to make it more extensible, how about making `\o` the option to change the output destination?

For example, `\o stdout` or `\o clipboard`.

Reply:
That's a awesome idea XD. Thank you. 
I fixed it.

I feel that the arguments stdout and clipboard are not exclusive, so I use `stdout` and `stdout_and_clipboard`.
（tuyoi kimotchi deha naidesu）